src/components/WebexMeetingControl/WebexMeetingControl.jsx

```diff
@@ -44,10 +44,17 @@ function renderButton(sc, action, display, style, showText, asItem, autoFocus, t
 
   if (asItem) {
     output = (
-      <div title={tooltip} style={style}>
+      <Button
+        type="ghost"
+        style={style}
+        size={24}
+        className={sc('collapsed-control-button')}
+        tooltip={tooltip}
+        verticalOffset={15}
```

Comment:
this is the vertical offset of the tooltip, right? as it's a property of the Button, it seems that it specifies the vertical offset of the button, not of the tooltip. Maybe `tooltipVerticalOffset`.